.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/*
+.DS_Store
```

Comment:
I have it on global .gitignore, but I find that other people tend not to, so it always manages to find it's way in. Same with Thumbs.db. But I'll just take it out if for the sake of consistency.